airbyte-scheduler/src/main/java/io/airbyte/scheduler/client/SchedulerJobClient.java

```diff
@@ -36,12 +36,6 @@
  */
 public interface SchedulerJobClient {
```

Comment:
not blocking: similar class doc comment here relating this and `SynchronousSchedulerClient`. will make it clear to future readers.

Reply:
added.